Makefile

```diff
@@ -4,109 +4,106 @@ DOCKER_PHP_CONTAINER_EXEC := $(DOCKER_COMPOSE) exec php
 DOCKER_DB_CONTAINER_EXEC := $(DOCKER_COMPOSE) exec mariadb
 DOCKER_PHP_EXECUTABLE_CMD := php -dmemory_limit=1G
 
-help:           ## Show this help
-    @fgrep -h "##" $(MAKEFILE_LIST) | fgrep -v fgrep | sed -e 's/\\$$//' | sed -e 's/##//'
+help: ## Show this help
+	@fgrep -h "##" $(MAKEFILE_LIST) | fgrep -v fgrep | sed -e 's/\\$$//' | sed -e 's/##//'
 
 addx:
 	chmod +x ./bin/install.sh
-
+	
 all:
 	make docker
 
 docker: ## Will migrate other tasks to the bash script too
 	make addx
 	./bin/install.sh
 
-start:          ## Start app
-    $(DOCKER_COMPOSE) up -d
-ifeq (,$(wildcard ./src/bb-config.php))
-    cp ./src/bb-config-sample.php ./src/bb-config.php
-    $(DOCKER_PHP_CONTAINER_EXEC) $(DOCKER_PHP_EXECUTABLE_CMD) ./bin/prepare.php
-endif
+start:	## Start app
+	$(DOCKER_COMPOSE) up -d
 
-start-recreate: ## Start app with full rebuild
+refresh: ## Start app with full rebuild
 	$(DOCKER_COMPOSE) up -d --build --force-recreate --remove-orphans
 
-stop:           ## Stop app
-    $(DOCKER_COMPOSE) stop
+stop:	## Stop app
+	$(DOCKER_COMPOSE) stop
 
-remove: stop    ## Stop and remove app
-    $(DOCKER_COMPOSE) rm -f
+remove:	stop ## Stop and remove app
+	$(DOCKER_COMPOSE) rm -f
 
-logs:           ## Show app logs
-    $(DOCKER_COMPOSE) logs -ft --tail=50
+logs: ## Show app logs
+	$(DOCKER_COMPOSE) logs -ft --tail=50
 
-exec-php:       ## Enter PHP container shell
-    $(DOCKER_PHP_CONTAINER_EXEC) bash
+exec-php: ## Enter PHP container shell
+	$(DOCKER_PHP_CONTAINER_EXEC) bash
 
-cache-flush:       ## Flush Twig cache
-    $(DOCKER_PHP_CONTAINER_EXEC) rm -fr src/bb-data/cache/*
+cache-flush: ## Flush Twig cache
+	$(DOCKER_PHP_CONTAINER_EXEC) rm -fr src/bb-data/cache/*
 
-exec-db:        ## Enter DB container shell
-    $(DOCKER_DB_CONTAINER_EXEC) bash
+exec-db: ## Enter DB container shell
+	$(DOCKER_DB_CONTAINER_EXEC) bash
 
-install: start  ## Install app after start
+install: start ## Install app after start
 	$(DOCKER_PHP_CONTAINER_EXEC) composer install --working-dir=src --no-progress --prefer-dist --no-dev
 	rm -rf ./src/install
 
-reinstall:      ## Reinstall app
-    rm -rf ./src/bb-config.php
-    make install
+reinstall: ## Reinstall app
+	rm -rf ./src/bb-config.php
+	make install
 
-test: start	## Run app tests
+test: start ## Run app tests
 	echo "Running unit tests"
 	echo > ./src/bb-data/log/application.log
 	echo > ./src/bb-data/log/php_error.log
 	rm -rf src/install
 	$(DOCKER_PHP_CONTAINER_EXEC) composer install --working-dir=src --no-progress --no-suggest --prefer-dist
 	$(DOCKER_PHP_CONTAINER_EXEC) ./src/vendor/bin/phpunit --dont-report-useless-tests ./tests/bb-modules/
 
-build:          ## Build App with Docker CI image
-    docker run --rm \
-        --mount type=bind,source=$(PWD),target=/opt -w /opt \
-        -e TRAVIS_TAG \
-        fordnox/docker-builder-ci \
-        make release
+build: ## Build App with Docker CI image
```

Comment:
Are we going to use Travis for CI or not.

If not we should remove it from make file

If yes we need to update the main image last time it was 2 years ago

Reply:
We ran into issues with Travis CI in BoxBilling and removed it. The leftovers can also be removed. I think GitHub Actions are enough for us now, if it's not, we can always set up CircleCI or something else in the future.